server.go

```
@@ -473,6 +474,13 @@ func (s *Server) handlerRoot(w http.ResponseWriter, r *http.Request) {
 	data.Movies = common.SortMoviesByName(movieList)
 	data.VotingEnabled, _ = s.data.GetCfgBool("VotingEnabled", DefaultVotingEnabled)
 
+	cycles, _ := s.data.GetPastCycles(0, 1)
```

Don't ignore the error, please.

---

fixed in 
2784fe1